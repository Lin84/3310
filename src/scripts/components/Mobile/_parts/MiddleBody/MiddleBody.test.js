import React from 'react';
import renderer from 'react-test-renderer';

// react component:
import MiddleBody from './index';

// props:
const removeLastEnteredData = () => {};
const handleClickSendButton = () => {};
const selectedValue = ['2', '3', '6'];

it('renders correctly', () => {
    const tree = renderer
        .create(<MiddleBody
            removeLastEnteredData={removeLastEnteredData}
            handleClickSendButton={handleClickSendButton}
            selectedValue={selectedValue.join('')}
        />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
