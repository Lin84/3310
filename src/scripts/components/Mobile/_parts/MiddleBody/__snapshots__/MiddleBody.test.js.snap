// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <button
    className="btn btn mobile__cancel-button"
    onClick={[Function]}
  >
    Cancel
  </button>,
  <div
    className="mobile__mini-display"
  >
    236
  </div>,
  <button
    className="btn btn mobile__send-button"
    onClick={[Function]}
  >
    Send
  </button>,
]
`;
